x-pack/metricbeat/module/aws/sqs/data.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package sqs
+
+import (
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schemaRequestFields = s.Schema{
+		"oldest_message_age": s.Object{
+			"sec": c.Float("ApproximateAgeOfOldestMessage"),
+		},
+		"messages": s.Object{
+			"delayed":     c.Float("ApproximateNumberOfMessagesDelayed"),
+			"not_visible": c.Float("ApproximateNumberOfMessagesNotVisible"),
+			"visible":     c.Float("ApproximateNumberOfMessagesVisible"),
+			"deleted":     c.Float("NumberOfMessagesDeleted"),
+			"received":    c.Float("NumberOfMessagesReceived"),
+			"sent":        c.Float("NumberOfMessagesSent"),
+		},
+		"empty_receives": c.Float("NumberOfEmptyReceives"),
+		"sent_message_size": s.Object{
+			"bytes": c.Float("SentMessageSize"),
```

Comment:
I'm still struggling with having `bytes` as a float. I understand it's because we get the average but should we just round it up or down? 

The main issue I see is that we use `scaled_floats` which can become inaccurate for large numbers if I remember correctly. So using long would scale better.